docs/installation.md

```diff
@@ -72,37 +72,50 @@ If you want to help develop Lychee, install the development dependencies by remo
 
 ### Configuration
 
-#### Public Directory
-After installing Lychee, configure your web server's root to be the `public` directory. The `index.php` in this directory serves as the interface for all HTTP requests to Lychee's API. Note that the `.env` file should **not** be exposed online as it contains the encryption key used for cookies as well as database credentials.
-
 #### Directory Permissions
-After installing Lychee, you need to configure some permissions. Directories within the `storage` and the `bootstrap/cache` directories should be writable by your web server or Lychee will not run. Additionaly `public/uploads`, `public/dist` and `public/sym` need to be writable to upload files, customise the CSS and utilise [symbolic links](https://lycheeorg.github.io/docs/settings.html#symbolic-link) respectively.
 
-#### Copy configuration file
-If you have not copied the `.env.example` file to a new file named `.env`, you should do that now.
+Directories within the `storage` and the `bootstrap/cache` directories must be writable by your web server or Lychee will not run. Additionally, `public/uploads`, `public/dist` and `public/sym` need to be writable to upload files, customise the CSS and utilise [symbolic links](https://lycheeorg.github.io/docs/settings.html#symbolic-link) respectively.
+
+#### Application Settings
 
-#### Application Key
-The next thing you should do after installing Lychee is set your application key to a random string. This is easily done by using the `php artisan key:generate` command.
+The main directory should contain a `.env` file.
+Normally, Composer will have copied this file for you while installing the dependencies.
+If the `.env` file does not exist, then you should copy `.env.example` to a new file named `.env` now.
+Note that the `.env` file should **not** be exposed online as it contains the encryption key used for cookies as well as database credentials.
 
-If you don't want to use this command and set the key yourself, this string should be 32 characters long and can be set in the `.env` environment file.
+##### Application Key
+
+The `.env` file contains the setting `APP_KEY` which holds your application key.
+Use the `php artisan key:generate` command to set the application key to a random value.
 
 **If the application key is not set, your user sessions and other encrypted data will not be secure!**
```

Comment:
I forget - will Lychee even run without it set?

Reply:
I was just wondering whether "will not be secure" is relevant if "Lychee will not work" applies? No strong feelings.

Reply:
(Just seen that's already updated!)